client/components/QuestionsAndAnswers/ModalForm.jsx

```diff
@@ -1,51 +1,238 @@
-      <Notes>
-        For privacy reasons, do not use your full name or email address
-      </Notes>
-    </Container1>
-    <Container1>
-      <LabelArea htmlFor="email">
-        Your Email
-      </LabelArea>
-      <InputArea
-        id="email"
-        type="email"
-        maxlength="60"
-        placeholder="Why did you like the product or not?"
+    </form>
+  );
+};
+
+export const AnswerForm = ({
+  newAnswer,
+  setNewAnswer,
+  setIsAdd,
+  handleAddAnswer,
+}) => {
+  const handleAnswerSumbit = (e) => {
+    e.preventDefault();
+    handleAddAnswer(newAnswer);
+    setIsAdd(false);
+  };
+
+  return (
+    <form>
+      <Container1>
+        <LabelArea htmlFor="yourAnswer">
+          Your Answer
+        </LabelArea>
+        <LargeText
+          id="yourAnswer"
+          type="text"
+          maxlength="1000"
+          onChange={(e) => setNewAnswer({ ...newAnswer, body: e.target.value })}
+        />
+      </Container1>
+      <Container1>
+        <LabelArea htmlFor="yourNickname">
+          What is your nickname
+        </LabelArea>
+        <InputArea
+          id="yourNickname"
+          type="text"
+          maxlength="60"
+          placeholder="Example: jack543!"
+          onChange={(e) => setNewAnswer({ ...newAnswer, answerer_name: e.target.value })}
+        />
+        <Notes>
+          For privacy reasons, do not use your full name or email address
+        </Notes>
+      </Container1>
+      <Container1>
+        <LabelArea htmlFor="yourEmail">
+          Your Email
+        </LabelArea>
+        <InputArea
+          id="yourEmail"
+          type="email"
+          maxlength="60"
+          placeholder="Example: jack@email.com"
+          onChange={(e) => setNewAnswer({ ...newAnswer, email: e.target.value })}
+        />
+        <Notes>
+          For authentication reasons, you will not be emailed
+        </Notes>
+      </Container1>
+      <Container1>
+        <LabelArea htmlFor="photos">
+          Upload your photos
+        </LabelArea>
+        <InputArea
+          id="photos"
+          type="file"
+          accept="image/*"
+          multiple
+        />
+      </Container1>
+      <SubmitInput
+        type="submit"
+        value="Submit Answer"
+        onClick={handleAnswerSumbit}
       />
-      <Notes>
-        For authentication reasons, you will not be emailed
-      </Notes>
-    </Container1>
-    <SubmitInput type="submit" value="Submit Question" />
-  </form>
-);
+    </form>
+  );
+};
 
+const ModalShadow = styled.div`
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 0px;
+  background-color: black;
+  opacity: 0.7;
+  z-index: 4;
+`;
+const Modal = styled.div`
+  max-width: 600px;
+  background-color: white;
+  position: fixed;
+  top: 75px;
+  z-index: 5;
+  max-height: calc(100% - 200px);
+  left: calc(50% - 250px);
+  display: flex;
+  flex-direction: column;
+  @media (max-width: 600px) {
+    left: 0px;
+    margin: 0px 10px;
+    padding: 10px;
+  }
+`;
```

Comment:
This comment applies to all styled-components in your module: consider moving your styled components into a separate `*.style.jsx` in a 'styles' folder. This is lower in priority than implementing functionality, but just something to consider later when refactoring. Jess and I went with adding our styles into a separate folder for code cleanliness and clearer folder organization.

Reply:
that's the plan for the weekend, haha, both of your layout looks great